README.md

```diff
@@ -52,6 +52,48 @@ For details regarding which type of OAuth grant will work best for your DocuSign
 
 For security purposes, DocuSign recommends using the [Authorization Code Grant](https://developers.docusign.com/esign-rest-api/guides/authentication/oauth2-code-grant) flow.
 
+## Tests
+### Test configuration
+1. Login with your DocuSign account on https://appdemo.docusign.com.
+2. Open Apps and Keys section in Settings and press Add App and Integration Key button.
+3. Enter App name, add secret key, generate RSA key and press Save button.
+4. Create new brand on https://admindemo.docusign.com/brands page and save its ID.
+5. Create new template on https://appdemo.docusign.com/templates page and save its ID.
+6. Create .env.example file copy with .env name and fill it with required data from steps 3-5.
+7. Write your generated private key to test\keys\docusign_private_key.txt file.
+
+Note: private key must be written to a file in the following format
+```
+-----BEGIN RSA PRIVATE KEY-----
+MIICXAIBAAKBgQCqGKukO1De7zhZj6+H0qtjTkVxwTCpvKe4eCZ0FPqri0cb2JZfXJ/DgYSF6vUpwmJG8wVQZKjeGcjDOL5UlsuusFncCzW.....
```

Comment:
As part of our company's security regulations we will need to remove any dummy secrets, IKs etc even if they are made up. Line 68 can be deleted here.